cmd/hauler/cli/store/add.go

```diff
@@ -62,40 +62,48 @@ func storeFile(ctx context.Context, s *store.Layout, fi v1alpha1.File) error {
 type AddImageOpts struct {
 	*RootOpts
 	Name string
+	Key  string
 }
 
 func (o *AddImageOpts) AddFlags(cmd *cobra.Command) {
 	f := cmd.Flags()
-	_ = f
+	f.StringVarP(&o.Key, "key", "k", "", "(Optional) Path to the key for digital signature verification")
```

Comment:
match other description references
```suggestion
	f.StringVarP(&o.Key, "key", "k", "", "(Optional) Path to the key for image signature verification")
```